variables.tf

```diff
@@ -216,6 +216,18 @@ variable "use_regional_endpoint" {
   description = "Whether to use a regional instead of the global endpoint address"
 }
 
+variable "subdomain_name" {
```

Comment:
In other places we called this just `subdomain` and I would place this above `use_regional_endpoint`.